lib/charms/mysql/v0/mysql.py

```diff
@@ -90,7 +90,7 @@ def wait_until_mysql_connection(self) -> None:
 
 # Increment this PATCH version before using `charmcraft publish-lib` or reset
 # to 0 if you are raising the major API version
-LIBPATCH = 26
+LIBPATCH = 28
```

Comment:
should this libpatch be 27 instead?

Reply:
I suspect CI will fail if you try to release LIBPATCH = 27 after merging this LIBPATCH = 28 (also, AFAIK charmcraft will refuse to upload the library with gap in LIBPATCH). I could be wrong.